src/ripple/json/impl/Writer.cpp

```diff
@@ -244,20 +243,20 @@ void Writer::output (std::string const& s)
 void Writer::output (Json::Value const& value)
 {
     impl_->markStarted();
-    writeJson (value, impl_->getOutput());
+    outputJson (value, impl_->getOutput());
 }
 
 template <>
 void Writer::output (float f)
 {
```

Comment:
@vinniefalco I'm not sure why vs is complaining. Clang doesn't complain. Not sure what's going on.